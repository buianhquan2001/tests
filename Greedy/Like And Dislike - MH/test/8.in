20
0 3 39
24 6 77
42 72 49
91 7 43
74 31 59
91 13 90
85 44 1
79 50 96
82 86 59
19 93 54
1 96 62
29 95 87
93 45 47
71 54 16
10 36 46
35 55 83
76 75 49
20 98 46
36 18 71
63 59 20
