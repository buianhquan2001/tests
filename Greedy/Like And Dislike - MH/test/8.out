Unknown
Unknown
Unknown
Like
Unknown
Unknown
Like
Unknown
Unknown
Dislike
Dislike
Unknown
Like
Like
Unknown
Unknown
Unknown
Dislike
Unknown
Unknown
